nixos/modules/services/web-apps/cloudlog.nix

```diff
@@ -502,13 +499,26 @@ in
           group = config.services.nginx.group;
         in
         [
-          "d ${cfg.dataDir}                0750 ${cfg.user} ${group} - -"
-          "d ${cfg.dataDir}/updates        0750 ${cfg.user} ${group} - -"
-          "d ${cfg.dataDir}/uploads        0750 ${cfg.user} ${group} - -"
-          "d ${cfg.dataDir}/backup         0750 ${cfg.user} ${group} - -"
-          "d ${cfg.dataDir}/logbook        0750 ${cfg.user} ${group} - -"
-          "d ${cfg.dataDir}/assets/json    0750 ${cfg.user} ${group} - -"
-          "d ${cfg.dataDir}/assets/qslcard 0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}                         0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/updates                 0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/uploads                 0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/backup                  0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/logbook                 0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/assets                  0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/assets/json             0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/assets/qslcard          0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/assets/sstvimages       0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/images                  0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/images/eqsl_card_images 0750 ${cfg.user} ${group} - -"
+          "C ${cfg.dataDir}/assets/json/dok.txt                              0640 ${cfg.user} ${group} - ${package}/assets/json.original/dok.txt"
+          "C ${cfg.dataDir}/assets/json/pota.txt                             0640 ${cfg.user} ${group} - ${package}/assets/json.original/pota.txt"
+          "C ${cfg.dataDir}/assets/json/satellite_data.json                  0640 ${cfg.user} ${group} - ${package}/assets/json.original/satellite_data.json"
+          "C ${cfg.dataDir}/assets/json/sota.txt                             0640 ${cfg.user} ${group} - ${package}/assets/json.original/sota.txt"
+          "C ${cfg.dataDir}/assets/json/US_counties.csv                      0640 ${cfg.user} ${group} - ${package}/assets/json.original/US_counties.csv"
+          "C ${cfg.dataDir}/assets/json/us_national_parksontheair.csv        0640 ${cfg.user} ${group} - ${package}/assets/json.original/us_national_parksontheair.csv"
+          "C ${cfg.dataDir}/assets/json/WABSquares.geojson                   0640 ${cfg.user} ${group} - ${package}/assets/json.original/WABSquares.geojson"
+          "C ${cfg.dataDir}/assets/json/wwff.txt                             0640 ${cfg.user} ${group} - ${package}/assets/json.original/wwff.txt"
+          "C+ ${cfg.dataDir}/assets/json/datatables_languages                0750 ${cfg.user} ${group} - ${package}/assets/json.original/datatables_languages"
```

Comment:
I'm using `C` to copy only if not exists, because many of those files have a feature in the app to update those data.